core/domain/collection_services.py

```diff
@@ -904,9 +1108,14 @@ def get_next_page_of_all_commits(
         page_size=feconf.COMMIT_LIST_PAGE_SIZE, urlsafe_start_cursor=None):
     """Returns a page of commits to all collections in reverse time order.
 
-    The return value is a triple (results, cursor, more) as described in
-    fetch_page() at:
-
+    Args:
+        page_size: int. Number of pages of commits to be returned.
+        urlsafe_start_cursor: str or None. If provided, the list of returned
+            commits starts from this datastore cursor. Otherwise, the returned
+            commits start from the beginning of the full list of commits.
+    Returns:
+        The return value is a triple (results, cursor, more) as described in
+        fetch_page() at:
```

Comment:
OK to link, but can we also do something more like this?

https://github.com/oppia/oppia/pull/2465/files#diff-e8e75b25b618133593df5ff9cd67a59bR228